content/en/platform/corda/5.0-dev-preview-2/developing/api/api-crypto-extensions.md

```diff
@@ -103,4 +103,4 @@ dependencies {
 You must reference `net.corda:corda-crypto-extensions`.
 {{< /note >}}
 
-For the CorDApp packaging instructions, see the relevant documentation.
+For the CorDApp packaging instructions, see the section on [CordApp Packaging](../tutorials/packaging.htm).
```

Comment:
That tutorial section isn't published for DP2

Reply:
Ah ok! Please revert that.

Reply:
OK reverted that change and pushed.